Doc/library/shelve.rst

```diff
@@ -117,7 +130,8 @@ Restrictions
   which can cause hard crashes when trying to read from the database.
 
 
-.. class:: Shelf(dict, protocol=None, writeback=False, keyencoding='utf-8')
+.. class:: Shelf(dict, protocol=None, writeback=False, keyencoding='utf-8', \
+                 serializer=None, deserializer=None)
```

Comment:
I think the *serializer* and *deserializer* parameters must be keyword-only, so to reflect the usage of `shelve.open`.